bin/lensInit.js

```diff
@@ -0,0 +1,35 @@
+#! /usr/bin/env node
+
+/**
+ * bin/lensInit.js
+ */
+
+const lensUtil = require('../src/lensinitUtils');
+const startTime = Date.now();
+const commander = require('commander');
+const validate = require('validate-npm-package-name');
+let lensName;
+
+commander.arguments('<lens-name>')
+  .action(name => {
+    if (name) {
+      lensName = name;
+    } else {
+      throw new Error('invalid lens name');
+    }
+  })
+  .parse(process.argv);
+
+const isValid = validate(lensName);
+
+if (isValid.validForNewPackages) {
+  const projectInit = `creating lens project ${lensName}`;
```

Comment:
I would capitalize `Creating` ... also, I think it might be useful to display the absolute path of the directory it will create, e.g. `Creating new lens "MyAwesomeLens" in ./users/jsmith/lenses`